storey/steps/assertion.py

```diff
@@ -0,0 +1,149 @@
+from dataclasses import dataclass
+from typing import Callable, List, Any, Collection
+
+from storey.dtypes import _termination_obj
+from storey.flow import Flow
+
+
+@dataclass
+class _Operator:
+    str: str
+    fn: Callable[[Any, Any], bool]
+
+    def __call__(self, x, y):
+        return self.fn(x, y)
+
+    def __str__(self):
+        return self.str
+
+
+_EQUALS = _Operator("==", lambda x, y: x == y)
+_NOT_EQUALS = _Operator("!=", lambda x, y: x != y)
+_ABOVE = _Operator(">", lambda x, y: x > y)
+_BELOW = _Operator("<", lambda x, y: x < y)
+_ABOVE_OR_EQUAL = _Operator(">=", lambda x, y: x >= y)
+_BELOW_OR_EQUAL = _Operator("<=", lambda x, y: x <= y)
+
+_ANY = _Operator("any of", lambda x, y: any([i for i in x if i in y]))
+_ALL = _Operator("all of", lambda x, y: len([i for i in x if i in x]) == len(y))
```

Comment:
Same here, but also it looks like one of the `x`s what suppose to be a `y`.

Reply:
Oh, and `i` implies that this is a counter. Better choose another letter to implie that it's a member.